jobConfig.example.yaml

```diff
@@ -82,21 +82,37 @@ jobs:
           request:
             jobParams.subject:
               type: string
-        - # Send an email. Requires email to be configured
+        - &email
+          # Send an email. Requires email to be configured
           actionType: email
+          ignoreErrors: true
           to: "{{{ job.contactEmail }}}"
-          subject: "[SciCat] {{ job.jobParams.subject }}"
-          bodyTemplateFile: demo_email.html
+          subject: |-
+            [SciCatLive] {{ default job.jobParams.subject "Job has been updated"}}
+          bodyTemplateFile: src/common/email-templates/job-template-simplified.html
     update:
       auth: "#jobOwnerUser"
-      actions: []
+      actions:
+        - # Only send emails for updates to status 'finished*'
```

Comment:
Should this be added to jobConfig.recommended.yaml as well, since sending email is such a common task?